__tests__/app.test.js

```diff
@@ -82,26 +82,48 @@ describe("4. GET /api/reviews/:review_id", () => {
 
 describe("5. GET /api/reviews", () => {
   test("Function to recieve an object of arrays in descending date order", () => {
-    return request(app).get("/api/reviews").expect(200).then((response)=>{
-      expect(response.body).toBeSortedBy("created_at",{descending:true})
-    })
-
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then((response) => {
+        console.log(response.body.review.rows);
```

Comment:
remove console.log :)

Reply:
Can't find that line in my VS code. Next to the file name in GitHub it says outdated. Could that be the reason?